src/docs/self-hosted/index.mdx

```diff
@@ -10,9 +10,35 @@ Our recommendation is to download the [latest release of the onpremise reposito
 
 ## Configuration
 
-You are very likely to want to change the default configuration for Sentry. Most, if not all, configuration options that you'd like to tweak are in `sentry/config.yml`, which is generated from [`sentry/config.example.yml`](https://github.com/getsentry/onpremise/blob/master/sentry/config.example.yml) at the time of installation. The file itself documents the most common configuration options as code comments, and you can find more about configuring Sentry over at <Link to="/config/">the configuration section of our developer documentation</Link>. You would likely need to change `system.url-prefix` (this is also prompted at the welcome screen, right after the installation) and settings starting with `mail.` even if we ship with a built-in SMTP server for basic stuff. GitHub and Slack integration settings also reside in this file. For more advanced things, you can have a look at [`sentry/sentry.conf.example.py`](https://github.com/getsentry/onpremise/blob/master/sentry/sentry.conf.example.py).
+You very likely will want to adjust the default configuration for Sentry. These facilities are available for that purpose:
 
-**Once you change your configuration, you'll need to restart all Sentry services by running `docker-compose restart web worker cron sentry-cleanup` (or just `docker-compose restart` to restart everything).**
+1. **`sentry/config.yml`**—Contains most, if not all, configuration options to adjust. This file is generated from [`sentry/config.example.yml`](https://github.com/getsentry/onpremise/blob/master/sentry/config.example.yml) at the time of installation. The file itself documents the most common configuration options as code comments. Some popular settings in this file include:
+
+    1. `system.url-prefix` (we prompt you to set this at the welcome screen, right after the installation)
+
+    1. `mail.*` (though we do ship with a basic SMTP server)
+
+    1. integrations for GitHub, Slack, etc.
+
+1. **`sentry/sentry.conf.py`**—Contains more advanced configuration. This file is generated from [`sentry/sentry.conf.example.py`](https://github.com/getsentry/onpremise/blob/master/sentry/sentry.conf.example.py) during installation.
+
+1. **Environment variables**—The available keys are defined in [.env](https://github.com/getsentry/onpremise/blob/master/.env). Use some system-dependent means of setting environment variables if you need to override any of them.
+
+1. [Geolocation](/self-hosted/geolocation/) uses **a custom configuration file** to conform to the underlying technology.
+
+You can find more about configuring Sentry at [the configuration section of our developer documentation](/config/).
+
+<Alert title="Note" level="info">
```

Comment:
May be add another `Alert` box, reminding people that we may update the example configurations across versions and they need to be manually applied. (we may also want to mention this in upgrading)

Reply:
Good idea! So good, we already have it! 😁 

<img width="770" alt="Screen Shot 2020-12-11 at 3 16 15 PM" src="https://user-images.githubusercontent.com/134455/101950800-dd284f00-3bc3-11eb-9b6f-f8212fe83df4.png">

Could convert to an `<Alert>`, but I think this is probably good for now, ya?

<img width="691" alt="Screen Shot 2020-12-11 at 3 21 52 PM" src="https://user-images.githubusercontent.com/134455/101951187-9d159c00-3bc4-11eb-82be-ee317238a03e.png">

Hopefully we can make some progress on this ☝️ sooner rather than later.